DOCS-12912-http-migration-kt (#1736)
mulesoft/docs-connectors: http/1.6/modules/ROOT/pages/http-about-http-connector-migration.adoc

:keywords: http, https, request, requester
:page-aliases: connectors::http/http-about-http-connector-migration.adoc

> The following list contains key changes for Anypoint Connector for HTTP (HTTP Connector) migration to Mule runtime engine (Mule) 4:

Reviewer: Suggestion: The Mule 4 version of Anypoint Connector for HTTP (HTTP Connector) contains the following key differences from the Mule 3 version of the connector: 
(Not sure if this conveys what you are trying to say here.)

* The `http:listener-config` element has a new inner connection element `http:listener-connection`.
* The `http:request-config` element has a new inner connection element `http:request-connection`.
* The HTTP body parameter replaces the source parameter in the request and is incorporated into the listener's response and error response.
* The HTTP *Listener* source and HTTP *Request* operation return attributes.
* HTTP parsing has been removed because DataWeave now manages special MIME types. Use DataWeave syntax to read and write payloads.
* The HTTP *Load static resource* operation XML code `static-resource-handler` is renamed to `load-static-resource`.
* To access HTTP listener and HTTP request attributes, the following DataWeave syntax replaces the Mule Expression Language (MEL) expressions for accessing past properties:
+
`#[attributes.<parameter>]`
+

> To get further information about mapping Mule 3.x MEL expressions to Mule 4 DataWeave expressions, refer to the xref:http-connector-xml-reference.adoc[HTTP Connector XML Reference] documentation.

Reviewer: ```suggestion
For more information about mapping Mule 3.x MEL expressions to Mule 4 DataWeave expressions, refer to the xref:http-connector-xml-reference.adoc[HTTP Connector XML Reference] documentation.
```
